src/chi/themes/brightspeed/_variables.scss

```diff
@@ -779,6 +779,8 @@ $carousel-dot-border-color:                 $color-border-black;
 $carousel-dot-hover-background-color:       $color-background-black;
 $carousel-dot-active-background-color:      $color-background-black;
 $carousel-dot-size:                         $icon-size-xs;
+$carousel-controller-height-dots:           calc(100% - #{$icon-size-xs});
+$carousel-controller-height-pagination:     calc(100% - #{$icon-size-sm2});
```

Comment:
@soujanreddy we should the same for the pagination height than the suggestion I did for dots. In this case, before that you should create a property in carousel.scss, in .chi-carousel__pagination of line-height: 1.25rem as a variable too. And then change $icon-size-sm2 by this var.

Reply:
@jllr Updated as per your comments. Please review.